{
  try {
    return method.invoke(sourceValue,arguments);
  }
 catch (  Exception e) {
    throw new KettleException("There was an error executing method [" + method.getName() + "] with "+ arguments.length+ " arguments",e);
  }
}
